apps/api-extractor/src/schemas/api-extractor.schema.json

```diff
@@ -164,6 +164,23 @@
       },
       "required": [ "enabled" ],
       "additionalProperties": false
+    },
+
+    "tsdocMetadata": {
+      "description": "Configures how the tsdoc metadata file will be generated",
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "description": "Whether to generate the tsdoc metadata file. The default is true.",
+          "type": "boolean"
+        },
+        "outputPath": {
```

Comment:
>outputPath [](start = 9, length = 10)

Consider renaming to `tsdocMetadataPath` so that we can more easily search for this field in the various places that refer to it